wallet/udb/upgrades_test.go

```diff
@@ -34,6 +35,7 @@ var dbUpgradeTests = [...]struct {
 	// No upgrade test for V7, it is a backwards-compatible upgrade
 	{verifyV8Upgrade, "v7.db.gz"},
 	// No upgrade test for V9, it is a fix for V8 and the previous test still applies
+	{verifyV10Upgrade, "v7.db.gz"},
```

Comment:
If we are testing a 9 -> 10 upgrade, we should have a version 9 database with the error.